### ISSUE TITLE
Foreman 1.8.0 release info on site eats up too much page space.

### ISSUE THREAD

**Issue by tmclaugh** [theforeman/theforeman.org#384]
Browsing the Foreman API is tough after the site redesign.  The 1.8.0 announcement takes up too much screen space.

Browser: Chrome 42.0.2311.90

URL: http://www.theforeman.org/api_v2.html

Screenshots:
- https://flic.kr/p/sAs6s9
- https://flic.kr/p/rW1VzA

**Comment by domcleal** [comment]
```
140811 < afisher> And keeps on taking up more room each time you click on a link! :)
140859 < afisher> With each new link it moves a bit further up the page.
```

**Comment by domcleal** [comment]
This has been fixed by #406.
